Directory.Build.props

```diff
@@ -3,8 +3,15 @@
     <HighEntropyVA>true</HighEntropyVA>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <TargetFramework>net6.0</TargetFramework>
-  </PropertyGroup>
-  <PropertyGroup>
+    
+    <Build_BuildNumber Condition="'$(Build_BuildNumber)' == ''">1.dev</Build_BuildNumber>
+    <AssemblyVersion>3.0.0.0</AssemblyVersion>
+    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionSuffix>preview$(Build_BuildNumber)</VersionSuffix>
+    <PackageVersion>$(VersionPrefix)-$(VersionSuffix)</PackageVersion>
+    <AssemblyFileVersion>$(Version)</AssemblyFileVersion>
```

Comment:
Do you have the commitID available somewhere in your pipeline? If you do,  you could add 

```xml
<InformationalVersion>$(AssemblyFileVersion)$(CommitIdForCurrentBuild)</InformationalVersion><InformationalVersion>$(AssemblyFileVersion)$(CommitIdForCurrentBuild)</InformationalVersion>
```

that would put the commidID right into the binary

Reply:
![image](https://user-images.githubusercontent.com/13396919/200061976-2346eee2-7f67-432a-a4fd-a39a3fa8161b.png)

Got that working, I figured `GenerateAssemblyFile` was turned off in this project, which was not generating this before.